specs/SystemCursorId.md

```diff
@@ -0,0 +1,122 @@
+by using IDC_* values and there is no easy way to convert the HCURSOR object we
+are returning into an IDC value after it's been created.
+
+This new API is to enable hosting applications like Office to get the current
+IDC value of the cursor instead of the HCURSOR as an option.
+
+Note that this is a COM and .NET spec as WinRT uses a separate enum structure to
+identify cursor IDs.
+
+
+# Description
+
+The `SystemCursorId` property will return the current system cursor ID reported
+by the underlying rendering engine for WebView2. It is not meant to return a
+value for any custom cursors, such as those defined by CSS.
+
+It can be used at any time but will only change after a `CursorChanged` event.
+
+Note that developers should generally use the `Cursor` property to support cases
+where there may be custom cursors.
+
+
+# Examples
+
+```cpp
+CHECK_FAILURE(m_compositionController->add_CursorChanged(
+    Callback<ICoreWebView2ExperimentalCursorChangedEventHandler>(
+        [this](ICoreWebView2ExperimentalCompositionController* sender, IUnknown* args)
+            -> HRESULT {
+            HRESULT hr = S_OK;
+            HCURSOR cursor;
+            UINT32 cursorId;
+            wil::com_ptr<ICoreWebView2ExperimentalCompositionController2> compositionController2 =
+                m_controller.query<ICoreWebView2ExperimentalCompositionController2>();
+            CHECK_FAILURE(compositionController2->get_SystemCursorId(&cursorId));
+            cursor = ::LoadCursor(nullptr, MAKEINTRESOURCE(cursorId));
+            if (cursor == nullptr)
+            {
+                hr = HRESULT_FROM_WIN32(GetLastError());
+            }
+
+            if (SUCCEEDED(hr))
+            {
+                SetClassLongPtr(
+                    m_appWindow->GetMainWindow() /* HWND */, GCLP_HCURSOR, (LONG_PTR)cursor);
+            }
+            return hr;
+        })
+        .Get(),
+    &m_cursorChangedToken));
+
+
+# API Notes
+
+See [API Details](#api-details) section below for API reference.
+
+
+# API Details
+
+```cpp
+/// This interface is continuation of the 
+/// ICoreWebView2CompositionController interface.
+[uuid(279ae616-b7cb-4946-8da3-dc853645d2ba), object, pointer_default(unique)]
+interface ICoreWebView2CompositionController2 : ICoreWebView2CompositionController {
+  /// The current system cursor ID reported by the underlying rendering engine
+  /// for WebView. For example, most of the time, when the cursor is over text,
+  /// this will return the int value for IDC_IBEAM. The systemCursorId is only
+  /// valid if the rendering engine reports a default Windows cursor resource
+  /// value. See:
+  /// https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-loadcursorw
+  /// Otherwise, if custom CSS cursors are being used, this will return 0.
+  /// To actually use systemCursorId in LoadCursor or LoadImage,
+  /// MAKEINTRESOURCE must be called on it first.
```

Comment:
Probably should say here and in C#

```cpp
/// If possible, use the Cursor property rather than the SystemCursorId property
/// because the Cursor property can express custom CSS cursors.
```